illumos-utils/src/running_zone.rs

```diff
@@ -234,7 +251,13 @@ impl RunningZone {
         ];
 
         for args in &commands {
-            running_zone.run_cmd(args)?;
+            if let Err(e) = running_zone.run_cmd(args) {
+                // `create-if` may fail if the vnic already has an interface.
+                // eat that error and continue.
+                if args[1] != "create-if" {
```

Comment:
This seems a little dicey in that it eats _any_ error from `create-if`. Any chance we could check that the error is specifically "this interface already exists"?

Reply:
Yeah, I realized after asking for a review that I left that bit in! See 676f760328b8fff79bc83b6f70b40f20040e9173 and 90cc69d2215debef8a475c65d93a2df98e3cf109

Reply:
Nice, those changes LGTM. Thanks!